src/index.spec.ts

```diff
@@ -70,65 +22,56 @@ describe('Browser sanity tests', () => {
           devServer = server;
           devServerPort = port;
         });
-
-        const puppeteerArgs = [];
-        if (process.env.CI === 'true') {
-            // tslint:disable-next-line:max-line-length
-            // https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-on-travis-ci
-            puppeteerArgs.push('--no-sandbox');
-            // https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#tips
-            puppeteerArgs.push('--disable-dev-shm-usage');
-        }
-
-        browser = await puppeteer.launch({
-            args: puppeteerArgs,
-            devtools: process.env.NODE_ENV === 'development',
-            executablePath: process.env.PUPPETEER_CHROME_PATH,
-        });
```

Comment:
we can't just delete all this, we need to move it into a `jest-puppeteer.config.js` which will get picked up by https://github.com/smooth-code/jest-puppeteer/blob/master/packages/jest-environment-puppeteer/src/readConfig.js#L32

Reply:
oh, it worked for me locally so I removed it. but okies 😄

Reply:
the `executablePath` is needed for docker